packages/p2p-v2/src/components/AdvertiserName/__tests__/AdvertiserName.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { APIProvider } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import AdvertiserName from '../AdvertiserName';
+
+const wrapper = ({ children }: { children: JSX.Element }) => <APIProvider>{children}</APIProvider>;
+let mockUseAdvertiserStats = {
+    data: {
+        fullName: 'Jane Doe',
+        name: 'Jane',
+        show_name: 0,
+    },
+    isLoading: true,
+};
+
+jest.mock('../../../hooks', () => ({
+    ...jest.requireActual('../../../hooks'),
+    useAdvertiserStats: jest.fn(() => mockUseAdvertiserStats),
+}));
+
+describe('AdvertiserNameStats', () => {
+    it('should render loading when data is not available', () => {
+        mockUseAdvertiserStats = {
```

Comment:
isn't the value same as the one in the above initialization?

Reply:
ah yeah no need to do this here